crates/runtime/src/pallets.rs

```diff
@@ -156,12 +155,9 @@ impl pallet_timestamp::Config for Runtime {
 parameter_types! {
     pub const UnsignedPriority: u64 = 1 << 20;
     pub const TransactionLongevity: u64 = u64::MAX;
-    pub const InvokeTxMaxNSteps: u32 = 1_000_000;
-    pub const ValidateMaxNSteps: u32 = 1_000_000;
```

Comment:
You can build custom chain config.
You will need to replace `﻿﻿VersionedConstants::latest_constants()`, there is only two calls to it.
It could be a follow up PR to add it back to the pallet config, but this time using the `VersionedConstants` type directly.

Reply:
I added it to #1567